src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/LicenceStatus.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence
+
+enum class LicenceStatus {
+  IN_PROGRESS,
+  SUBMITTED,
+  APPROVED,
+  ACTIVE,
+  REJECTED,
+  INACTIVE,
+  VARIATION_IN_PROGRESS,
+  VARIATION_SUBMITTED,
+  VARIATION_APPROVED,
+  ;
+}
```

Comment:
Rather than use the other set of statuses with some unimplemented statuses, just exposing what we use.